tests/page_objects/care_platform_page.yaml

```diff
@@ -539,3 +539,5 @@ care_platform_edit_participants_vendor:
   modal_title_label: //div[@id='modals']//h4[text()='$AND_CLI_MODAL_NAME$']
   dialog_title_label: //div[@id='dialogs']//h4[text()='$AND_CLI_MODAL_NAME$']
   modal: //div[@id='modals']//div[contains(@class,'modal')]
+  nav_vendor_button: //ul[@class='form-navigation']//li/button[text()='Vendors']
```

Comment:
The navigation button we add it to "care_platform_edit_participant_records", could you move it over there to maintain the convention

Reply:
I dit it in this commit https://github.com/neveralone-chs-org/TestRay/pull/161/commits/88322936bc648565269208b8d157882b99abd196